users/alfrdmalr/config.h

```diff
@@ -1,13 +1,7 @@
 #pragma once
 
 #ifdef AUDIO_ENABLE
-    #define STARTUP_SONG SONG(PREONIC_SOUND)
-    // #define STARTUP_SONG SONG(NO_SOUND)
-
-    #define DEFAULT_LAYER_SONGS { SONG(QWERTY_SOUND), \
-                                  SONG(COLEMAK_SOUND), \
-                                  SONG(DVORAK_SOUND) \
-                                }
+  //audio-specific code
 #endif
```

Comment:
This block can be removed altogether.

Reply:
I was planning to add song definitions in, but I'll remove the block until I actually do so. Thanks.